clientv2/src/components/ProfileViewer/options.ts

```diff
@@ -0,0 +1,311 @@
+export const institutionOptions = [
```

Comment:
where did you get this list from?

Reply:
It's just a list of common institutions for the auto complete. It's not the only options.

Reply:
ok, perfect